cmd/demo/demo.go

```diff
@@ -35,6 +40,24 @@ To execute commands inside the demo container, use exec:
 `
 )
 
+// Long descriptions are whitespace sensitive when generating docs using sphinx.
+const (
+	initShort = `Download the Flyte sandbox image, create local state folder and place a default config file in it`
+	initLong  = `
+Flyte Demo	
+When you run::
+::
+
+ flytectl demo init  
+
+flytectl will ensure you have the latest run time image, create a local state directory for you if not present,
```

Comment:
can we explain here whats the purpose of the state directory. Is it just for default flyte config file and link to example config